spec/features/content_timeline_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.feature 'Content Timeline', type: :feature do
+  let!(:organization) { create(:organization) }
+  let!(:user) { create(:user, email: 'jesse@mountainmantechnologies.com', organization: organization) }
+  let!(:content) { create(:gallery, organization: organization, title: 'Albuquerque') }
+
+  with_versioning do
+    it 'add record to content timeline' do
+      visit '/'
+      click_link "Sign In"
+      fill_in 'Email', with: user.email
+      fill_in 'Password', with: 'passwd'
+      click_button 'Login'
+      click_link 'My Account'
+      expect(page).not_to have_content('Lubomir was here')
+      click_link 'Content'
+      expect(page).to have_content('Albuquerque')
+      click_link 'Albuquerque'
+      click_link 'Edit'
+      fill_in 'Title', with: 'Lubomir was here'
+      click_button 'Update Content'
+      content.reload
+      click_link 'Dashboard'
+      expect(page).to have_content(:all, 'Lubomir was here')
+    end
```

Comment:
notes on this:
1. this is feature test, use `background` and `scenario`
  event though rspec is one ecosystem, and regular unit test helpers are working here, it does not mean that it suit to use them everywhere

2. give a thought on the scenario steps, they are chaotic, align them in one chain in order to get a clear vision of use-case

3. don't try to cover all things in 1 feature test as "it covers everything", it's not working like that and will only make the coverage difficult to understand and make corrections in future. Make few smaller unit tests to cover complicated places.

note: just making method private so it'd get excluded from coverage trace does not mean that it should not be tested, feature tests are just invoke such methods and they appear covered, however they do not ensure that that specific use case covers all possible use-cases

Reply:
Roman it's ok here. let's focus more on logic it's now more important

Reply:
ps: it was Igor :)